invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/SeriesRelations.js

```diff
@@ -287,16 +268,11 @@ class SeriesRelationsTabPanel extends Component {
     return selections;
   }
 
-  getTabRows(relation, pickColumns) {
+  getTabRows(relation) {
     const rows = [];
     if (!this.props.relations[relation]) return [];
-
     for (const obj of this.props.relations[relation]) {
```

Comment:
This loop is not necessary. Maybe the whole function `getTabRows` could be removed too and just use the value in `this.props.relations[relation]`.